docs/build-on-linea/linea-sdk/index.md

```diff
@@ -27,8 +27,44 @@ The SDK focuses on interacting with smart contracts on both Ethereum and Linea n
 
  5. Claiming messages (use one of the get message methods to grab all the parameters values)
 
+ 
+:::info[New Features]
 
-The code example below demonstrates how to initialize the SDK with the relevant configuration options, and how to call the featured methods.
+ The updated Linea SDK package enhances the L1 contract interaction and adds support for the new L1 claiming system, which is based on a merkle tree.
+ 
+ There are two important things to note:
```

Comment:
An additional point can be made here as well (so there are "three important things to note")

The Previous L1 claiming function and code samples provided here caters for the transition period where pre-transition messages are claimed without the Merkle proof and post-transition with proof. If this SDK is being consumed after the transition, using the logic that switches between Merkle and non-Merkle proof claiming is sub-optimal.

Reply:
added